Data_Collection/CRON_scripts/colombia_data.py

```diff
@@ -33,14 +33,9 @@ def chunked_fetch(start, chunk_size, maxrecords):
         # Fetch the set of records starting at 'start'
         # create a df with this chunk files
         df_new = pd.DataFrame.from_records(client.get("gt2j-8ykr", offset=start, limit=chunk_size,
-                                    order = 'fecha_reporte_web', where = f'fecha_reporte_web > "{slice_date}"')
-        
-        if df_new.empty:
```

Comment:
@eduardocorrearaujo there is syntax error here. Actually it's in the line above, it is missing a closing parenthesis. That's why when this code get transfered to the epigraphhub library, it should have unit tests.